roles/portainer/tasks/main.yml

```diff
@@ -27,7 +27,9 @@
           traefik.http.routers.portainer.tls.certresolver: "letsencrypt"
           traefik.http.routers.portainer.tls.domains[0].main: "{{ ansible_nas_domain }}"
           traefik.http.routers.portainer.tls.domains[0].sans: "*.{{ ansible_nas_domain }}"
-          traefik.http.services.portainer.loadbalancer.server.port: "9443"
+          traefik.http.services.portainer.loadbalancer.server.port: "{{ portainer_port }}"
```

Comment:
This is incorrect - the server port should be the port the container listens on, not the port the user chooses to expose it on.

Reply:
Fixed.